src/titiler/extensions/titiler/extensions/render.py

```diff
@@ -0,0 +1,123 @@
+"""render Extension."""
```

Comment:
it might be fine to move this into `stac.py`

Reply:
I think the `stac.py` extension serves a very different purpose - to create a STAC item based on an incoming COG. Even though both deal with STAC, the functionality/purpose of the two does not seem related to me.